### PR TITLE
For technical review gsq-features

### PR DESCRIPTION
**Pull request by johnmckellar**

### CODE REVIEW

File: vocabularies/gsq-features.ttl [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    dct:source "NACSN, 1983, 2005; Kumpulainen, R.A., 2017." ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Lithodemic feature"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:permit a skos:Concept ;
+    skos:definition "In general mineral or energy rights in Australia are reserved to the Crown. The acquisition of rights to minerals or energy stems from legislative frameworks in Queensland. These frameworks provide initially for exploration of the resource, and consist of the grant by the Crown in the form of exploration permits, licences or leases. Exploration permits, leases or licences permit works to be undertaken to determine the likely existence of minerals or resources. Actual mining is subject to a further grant of mining or minerals production leases or licences. The legislation also provides for the payment of royalties to the State and to compensate the owners or occupiers of the surface land."@en ;
```
Comment (KellyVance) [inline vocabularies/gsq-features.ttl]: "In general, mineral or energy rights..........."
comma needed

File: vocabularies/gsq-features.ttl [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:placeName a skos:Concept ;
+    skos:definition "Place Names are names given to natural and artificial geospatial features, such as administrative areas, political regions, mountain ranges, rivers, bays etc."@en ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Place name"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:resourceAccumulation a skos:Concept ;
+    skos:definition "Resource accumulations are naturally occurring accumulations or concentrations of metals, minerals or hydrocarbons of sufficient size and concentration that might, under favourable circumstances, have economic value. Economic concentrations of metals or other mineral commodities are known as ore."@en ;
```
Comment (KellyVance) [inline vocabularies/gsq-features.ttl]: "metals, minerals or hydrocarbons".
Should we include quarry resources here too? Is coal implicit in minerals?

"Resource accumulations are naturally occurring accumulations or concentrations of metals, minerals, coal, hydrocarbons, or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore."

Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: What about the following?

"Resource accumulations are naturally occurring accumulations or concentrations of minerals (including metals and coal), hydrocarbons, or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore."

Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: Forgot about quarrying:

"Resource accumulations are naturally occurring accumulations or concentrations of minerals (including metals and coal), hydrocarbons, quarrying resources or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore.

Comment (KellyVance) [reply vocabularies/gsq-features.ttl]: I like the definition by @johnmckellar

**Comment by johnmckellar** [comment]
David,

Approved by Vance. I have now added you.

**Pull request merged**